content/02.main-text.md

```diff
@@ -29,20 +29,20 @@ Existing platforms work well for editing text and are widely used for scholarly
 However, they lack features that are important for open writing, such as managing a collaborative manuscript and attributing precise credit among many authors (Table @tbl:platforms).
 None of these platforms offer the ability to address thematically related changes together and enable multiple authors to iteratively refine proposed changes.
 
-| Feature | Manubot | Authorea + BibTeX | Overleaf v2 + BibTeX | Google Docs + Paperpile | Word Online<sup>1</sup> | Markdown on GitHub |
+| Feature | Manubot | Authorea | Overleaf v2 | Google Docs + Paperpile | Word Online<sup>1</sup> | Markdown on GitHub |
 |---|---|---|---|---|---|---|
 | Multi-author editing | Yes | Yes | Yes | Yes | Yes | Yes |
 | Propose changes | Yes | No | No | Yes | No | Yes |
 | Continuous integration testing | Yes | No | No | No | No | No |
 | Multi-participant conversation for changes | Yes | No<sup>2</sup>  | No<sup>2</sup> | No<sup>2</sup> | No<sup>2</sup> | Yes |
-| Character-level provenance for text | Yes | No (versions tracked by day) | No<sup>3</sup> | Requires manual inspection of history | Not after changes are accepted | Yes |
+| Character-level provenance for text | Yes | Yes | No<sup>3</sup> | Requires manual inspection of history | Not after changes are accepted | Yes |
 | Bibliography management | Yes | Yes | Yes | Yes | No, requires the Word desktop application | No |
-| Citation by identifier | Yes | No | No | No | No | No |
+| Citation by identifier | Yes | Yes | No | No | No | No |
 | Editing software | Any text editor | Web interface | Web interface | Web interface | Web interface | Any text editor |
-| Document format | Markdown | LaTeX | LaTeX | Proprietary | Proprietary | Markdown |
+| Document format | Markdown | HTML | LaTeX | Proprietary | Proprietary | Markdown |
```

Comment:
It seems like the primary output format is HTML although the input is a mix of HTML exposed via a WYSIWIG-editor, markdown cells and latex cells.

Reply:
Agree